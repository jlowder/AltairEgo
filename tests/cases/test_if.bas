10 INPUT "NUMBER";N
20 IF N>8 THEN PRINT "TOO BIG":GOTO 10
30 PRINT "OK, NUMBER IS";N
40 END
