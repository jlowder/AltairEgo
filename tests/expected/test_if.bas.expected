Altair Ego: Emulating Altair BASIC 32K Rev. 3.2
OK
OK
NUMBER? OK, NUMBER IS 0
OK
